core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -953,6 +953,9 @@ private Object readResolve() {
     @Nonnull
     @Restricted(NoExternalUse.class)
     public InstallState getInstallState() {
+        if (installState == null || installState.name() == null) {
+            return InstallState.UNKNOWN;
```

Comment:
I think it's irrelevant, it's been corrupted (by XStream or some other spot), corrupt state is an unknown state.